### ISSUE TITLE
Tiedtke convection ledger: closure dimension, momentum transport, phase-keyed latent heat, surface layer

### ISSUE THREAD

**Issue by duncanwp** [climate-analytics-lab/jax-gcm#676]
Found by a re-audit of the July 2026 ECHAM+RRTMGP review against `dev` @ `f6d1bcd`. Parent: #614.

Four residual deviations in the Tiedtke–Nordeng tendency assembly, after the campaign's `cudtdq` ledger fix. Grouped because they are one pass over `flux_tendencies.py`.

## 1. The CAPE closure fallback is dimensionally a velocity

`flux_tendencies.py:467` and `:535-542` (`mass_flux_closure_blend`, the live path) form `mf_cape = cape / (c.grav * tau)` — units m/s, not kg m⁻² s⁻¹.

Nordeng's rescale *has* landed (`tiedtke_nordeng.py:751-807`, `zmfub1 = zcape·zmfub/(zheat·tau)`, matching `mo_cumastr.f90:901`), and since the deep amplitude is linear in the first guess the dimensional error largely cancels **except** where a clip binds — and both `mf_deep` and `zmfub1` are hard-clipped. For shallow/mid (`conv_type != 1`) `rescale == 1`, so the velocity-dimension closure *is* the final flux with no correction at all.

Both references divide CAPE by τ times a **CAPE-consumption rate per unit cloud-base mass flux**, not by `g`:
- ECHAM's deep first guess is a moisture-convergence closure with a constant fallback: `mo_cumastr.f90:567` `zmfub = FSEL(-zlo1, 0.01_wp, zdqpbl/(grav*MAX(zqumqe,zdqmin)))` — the literal `0.01 kg m⁻² s⁻¹` is what ECHAM uses when PBL moisture convergence is non-positive.
- CAM's ZM is `mb = ΔCAPE/(tau·dadt)` (`zm_convr.F90:2477-2481`) where `dadt` has units 1/mb·1/s, giving kg m⁻² s⁻¹. jcm substitutes `g` [m s⁻²] for that rate — it resembles ZM's shape but is dimensionally invalid under either reference.

Fix: replace the fallbacks with ECHAM's constant `zmfub = 0.01` (clipped by `zmfmax`) and let the existing Nordeng rescale supply the CAPE dependence for deep. For shallow/mid, extend the `use_moisture` branch — ECHAM uses the same `zdqpbl/(g·Δq)` budget for ktype 1 *and* 2. Note this couples to the `tau`-gradient item in the clips/floors issue: the same clips are what zero `d(precip)/d(tau)`.

## 2. Momentum transport is not `cududv` — and escapes the stability cap

`flux_tendencies.py:360-380` now has ECHAM's deviation-flux form and the `ik = jk−1` upstream offset (the invented "PGF relaxation" is gone — good). Remaining gaps vs `mo_cufluxdts.f90:895-951`:

- **No prognostic plume wind.** ECHAM builds `puu`/`pvu` in `cuasc`; jcm substitutes the constant cloud-base wind.
- **No separate downdraft wind.** ECHAM has `zmfdu = pmfd*(pud − puen)`; jcm folds `mfd` into `net_mf` and gives the downdraft the *cloud-base* wind, so downdraft momentum transport has the wrong sign structure.
- **No sub-cloud extension.** ECHAM tapers with `zzp = ((p_sfc−p)/(p_sfc−p_cbot))` (squared for ktype 3) so cumulus friction acts through the sub-cloud layer; jcm's `conv_mask` (`:401`) zeroes `dudt`/`dvdt` for `k > kbase` entirely.
- **No surface-layer term.** ECHAM has an explicit `jk == klev` branch; jcm writes the `(nlev-1,)` divergence into `[:-1]`, so `dudt[nlev-1] == 0` always.

**Fix the last point first — it is one line and it is a self-inconsistency, not a fidelity gap.** `tiedtke_nordeng.py:1144-1145` passes `dudt`/`dvdt` **unscaled** while T, q, qc, qi, precip and the mass fluxes all get `cap_scale`. When the `_DTDT_MAX` cap binds — which is routine, not exceptional — momentum is delivered at full strength while the thermodynamics derived from the *same* mass flux is scaled to ~9 %. That breaks the proportionality the cap's own comment (`:1128-1131`) claims preserves column conservation. Neither reference has such a cap, so there is nothing to be faithful to.

For reference, CAM's `zm_conv_momtran.F90` has *more* machinery than ECHAM's `cududv`, not less — prognostic in-cloud wind (`:285-289`), separate downdraft wind (`:303-307`), a convective PGF term `pgu = -momcu*0.5*mududp` with `momcu = 0.4` (`:239`), and a Boville–Bretherton KE-dissipation heating (`:399-420`). So there is no "CAM does it more simply" defence.

## 3. The condensate-flux heating term hardcodes L_v

`flux_tendencies.py:300`:
```python
dtedt_k_levels = (dse_flux_div - c.alhc * pmful_div) / (c.cpd * layer_mass_per_area)
```
ECHAM's `cudtdq` uses the phase-keyed half-level `palvsh`. The per-level source term 44 lines below (`:344`, `zalv = where(T > tmelt, alhc, alhs)`) **is** phase-keyed, so the two halves of one ledger disagree by ~12 % wherever `T < tmelt`.

Fix: replace `c.alhc * pmful_div` with a phase-keyed `jnp.diff(zalv_half * lu * mfu)` using the same `where` as `:344`.

Related, in dead code: `adjustment.saturation_adjustment` (`:143-155`) still applies an `(alhs−alhc)` ice-fraction ramp on top of a Newton step that *already* uses `alhs` below freezing (`:81`), so the effective latent heat is `alhs + frac_ice·alhf` — **+6.7 % at 260 K, +11.8 % below 250 K** — and its comment claiming to "correct for the ice-fraction difference" is now false. Nothing on the model path calls it (campaign PR 4 removed the grid-mean adjustment), but it is exported. Delete both it and `convective_adjustment`, or fix the double correction.

## 4. The surface layer never receives a convective tendency

`flux_tendencies.py:403-408` builds `jnp.zeros(nlev).at[:-1].set(dtedt_k_levels)` — the divergence array is `(nlev-1,)` and index `nlev-1` is never written. Verified on the tropical sounding: `tendencies.dtedt[46] == 0.0` and `dqdt[46] == 0.0` exactly, while `dtedt[45] = 1.15e-3 K/s`. ECHAM's `cudtdq` has an explicit `jk == klev` branch.

Separately the divergence is still truncated at cloud base (`conv_mask` with `cloud_bottom = kbase`), so downdraft flux divergence below cloud base is discarded even though `mfd` is designed to descend to the surface with the `itopde` taper. ECHAM's loop runs `ktopm2..klev` with no such truncation. On the probe column this cost only 1.1e-8 W/m² because the downdraft died at its LFS, so the magnitude is unverified — but the truncation breaks the telescoping of the divergence sum whenever `mfd ≠ 0` at `kbase`.

Fix: extend the divergence to `nlev` entries with the ECHAM `klev` closure (surface half-level flux is zero, so `div[nlev-1] = −F[nlev-1]/mass`), and raise `cloud_bottom` to `nlev-1` — the terms are already zero wherever `mfu = mfd = 0`. Adjacent to but distinct from #530.

## Also resolved, recorded for closure

`energy_conservation_check` (`adjustment.py:166-214`) is anti-correlated with actual conservation: a conserving adjustment returns `L·dqc ≠ 0`, while `dT=0, dq=−x, dqc=+x` (condensation with no latent heating) returns exactly 0 and passes. Units are mixed (W/kg vs W/m², no mass weighting) and the tests pin nothing (`|imbalance| < 50.0` on a case whose value is 0.42). Dead code — only `adjustment_test.py` calls it — but exported. Delete it, or rewrite as the mass-weighted moist-static-energy residual with a test that fails on a deliberately non-conserving state.

## Refs

Review findings 2.5, 2.6d, 2.7, 2.8b, 2.8c, 2.8d. Related: #530.

**Comment by duncanwp** [comment]
One more ledger gap for this issue's list, found while porting the zdqcv split (#699): ECHAM's SHALLOW re-closure (mo_cumastr.f90:909-936) is not ported. After the downdrafts, ktype=2 columns recompute the cloud-base flux from the PBL moisture budget *including the downdraft moisture* (`zqumqe = pqu + plu − zeps·zqd − (1−zeps)·zqenh`) and apply it only when it moves less than 20 % from the first guess. jcm applies the pre-downdraft moisture-anchored flux with no re-closure. With most of the globe now classified shallow under the faithful split, this term matters more than it did under the old proxies.

**Comment by duncanwp** [comment]
One more ledger item, found writing the #641 science doc (PR #771): when the 5 K/hr `_DTDT_MAX` safety cap fires, `TiedtkeConvection.__call__` rescales T/q/qc/qi/precip and the mass fluxes by `cap_scale` but returns `dudt`/`dvdt` unscaled — a capped plume's momentum transport keeps full amplitude. Whichever way the momentum-transport item here resolves, the cap should treat momentum consistently with the rest of the ledger.
